### ISSUE TITLE
Use six for iteritems

### ISSUE THREAD

**Issue by tdenewiler** [sscpac/statick#96]
When working on an external plugin for Statick I ran the self check and got the following error using the installed version of Statick (v0.2.4):

```
Traceback (most recent call last):
  File "/usr/local/bin/statick", line 39, in <module>
    main()
  File "/usr/local/bin/statick", line 26, in main
    issues, success = statick.run(path, parsed_args)
  File "/usr/local/lib/python3.5/dist-packages/statick_tool/statick.py", line 254, in run
    plugin.report(package, issues, level)
  File "/usr/local/lib/python3.5/dist-packages/statick_tool/plugins/reporting/write_file_reporting_plugin.py", line 42, in report
    for _, value in issues.iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```

This is a problem related to Python 2/3 compatibility, since `iteritems` was removed in Python 3. For more discussion see:

https://stackoverflow.com/questions/30418481/error-dict-object-has-no-attribute-iteritems?rq=1

Grepping through the `statick_tool` directory there are two existing spots where `iteritems` is used. Those should be updated to use one of the Python 2/3 compatibility tools. Previously I used `six` in `statick_tool/statick.py`.

**Comment by creffett** [comment]
I'm surprised that I didn't catch those earlier. In other places, I used the list(dict.items()) idiom, which apparently is a performance hit but works with both Python2/Python3.

**Comment by tdenewiler** [comment]
I merged in a fix for this in #97.

I'm fine with the current state of supporting Python 2/3 for `iteritems`. Would you prefer we make changes to be consistent in the approach, i.e., switch to using all `six` or all `list(dict.items())`?
